src/api/init.ts

```diff
@@ -155,10 +155,20 @@ export async function startApiServer(opts: {
       format: logger.format,
       transports: logger.transports,
       metaField: (null as unknown) as string,
-      statusLevels: {
-        error: 'error',
-        warn: httpLogLevel ?? 'http',
-        success: httpLogLevel ?? 'http',
+      statusLevels: false,
+      level: function (req, res) {
+        let level = '';
+        if (res.statusCode >= 100) {
+          level = 'info';
+        }
```

Comment:
I think you could simplify this by setting this as `info` initially:
```suggestion
        let level = 'info';
```